content/docs/csidriver/features/powerflex.md

```diff
@@ -661,4 +661,66 @@ allowedTopologies:
         values:
           - csi-vxflexos.dellemc.com
 ```
-Once the volume gets created, the ControllerPublishVolume will set the QoS limits for the volumes mapped to SDC.
\ No newline at end of file
+Once the volume gets created, the ControllerPublishVolume will set the QoS limits for the volumes mapped to SDC.
+
+## Rename SDC
+
+Starting in version 2.6, the CSI Driver for PowerFlex now supports renaming of SDC. To use this feature, the node section of values.yaml should have renameSDC keys enabled with a prefix value.
+
+If you want to rename the SDC, please make the following edits to your values file at `csi-vxflexos/helm/csi-vxflexos/values.yaml:`
+```yaml
+# "node" allows to configure node specific parameters
+node:
+   ...
+   ...
+
+  # "renameSDC" defines the rename operation for SDC
+  # Default value: None
+  renameSDC:
+    # enabled: Enable/Disable rename of SDC
+    # Allowed values:
+    #   true: enable renaming
+    #   false: disable renaming
+    # Default value: "false"
+    enabled: false
+    # "prefix" defines a string for the new name of the SDC.
+    # "prefix" + "worker_node_hostname" should not exceed 31 chars.
+    # Default value: none
+    # Examples: "rhel-sdc", "sdc-test"
+    prefix: "sdc-test"
+```
+The renameSDC section is going to be used by the Node Service, it has two keys enabled and prefix:
+* `enabled`: Boolean variable that specifies if the renaming for SDC is to be carried out or not. Is true then the driver will perform the rename operation. By default, its value will be false.
```

Comment:
correct the typo from Is True to If True

Reply:
done